spinalcordtoolbox/reports/qc.py

```diff
@@ -605,8 +727,8 @@ def layout(qslice):
         print("WARNING! Platform undetectable.")
 
 
-def generate_qc(fname_in1, fname_in2=None, fname_seg=None, angle_line=None, args=None, path_qc=None, dataset=None,
-                subject=None, path_img=None, process=None):
+def generate_qc(fname_in1, fname_in2=None, fname_seg=None, angle_line=None, args=None, path_qc=None, fps=None,
+                dataset=None, subject=None, path_img=None, process=None):
```

Comment:
Nitpick, but it might be safer to add `fps` as the last parameter, just in case there are any calls to `generate_qc` passing parameters positionally.